posthog/hogql/database/database.py

```diff
@@ -399,6 +399,7 @@ def create_hogql_database(
     team: Optional["Team"] = None,
     modifiers: Optional[HogQLQueryModifiers] = None,
     timings: Optional[HogQLTimings] = None,
+    skip_dw_tables: bool = False,
```

Comment:
We usualy abbreviate it with `dwh` rather than `dw`. Also. Because this isn't used often, I'm in favor of *not* abbreviating it at all

```suggestion
    skip_data_warehouse_entities: bool = False,
```

Also, calling `tables` is not accurate. There are also views and joins. Maybe `entities`? Maybe `entries`? Anyways, hardest problem in computer science, blahblahblah